modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Deleting content within selections that contained content outside a table and content within a cell could result in the entire contents of the cell being incorrectly deleted #TINY-7596
```

Comment:
I need help with explaining this bug in shortform changelog format

Reply:
I like your second one, although it's more 'incorrect' content because it would also not delete any of the content outside the table. Anyone else have any thoughts?

Reply:
```suggestion
- Delete operations could behave incorrectly if the selection crossed a table boundary #TINY-7596
```